fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -563,10 +563,36 @@ public PartitionInfo getPartitionInfo() {
         return partitionInfo;
     }
 
+    public List<String> getPartitionColumnNames() {
+        List<String> partitionColumnNames = Lists.newArrayList();
+        if (partitionInfo instanceof SinglePartitionInfo) {
+            return partitionColumnNames;
```

Comment:
partitionColumnNames has upperCase？

Reply:
It can but we need to check it ignore case